### ISSUE TITLE
Note for non-dynamic addons

### ISSUE THREAD

**Issue by Samq64** [ScratchAddons/ScratchAddons#5646]
### Why this would be helpful

Now that most addons can be dynamically enabled users might not think of reloading the tab when enabling addons and think it doesn't work.

### My suggestion

Automaticlly adding a note to all addons that don't support dynamic enable letting the user know they'll need to reload the tab for changes to take effect.

### Possible alternatives

Making all addons dynamic or adding a button to reload all Scratch tabs like TurboWarp does.

### Additional context

Related: #1673, #5181

**Comment by WorldLanguages** [comment]
I think the note/modal should only be visible after the user enables or disables an addon.

**Comment by iqnite** [comment]
Or a "Reload required" tag next to the addon's name, like "Recommended" and "Dangerous".

**Comment by BroJac5246** [comment]
> 
> <img width="111" alt="image" src="https://github.com/ScratchAddons/ScratchAddons/assets/50552317/acb17f1b-533e-4d58-b678-146b0a6a0583">
> Why the downvotes? It makes more sense than the darn "Forum" tag!

Because it doesn't fit with the other tags, unlike the "Forum" tag, plus it'll confuse users.

**Comment by WorldLanguages** [comment]
> <img alt="image" width="111" src="https://user-images.githubusercontent.com/50552317/238328151-acb17f1b-533e-4d58-b678-146b0a6a0583.png"> Why the downvotes? It makes more sense than the darn "Forum" tag!

Because it's not necessary to tell the user that a reload is required until they actually toggle the addon or change a setting. And after they do that, some type of modal would be better than adding a tag.

**Comment by iqnite** [comment]
> > <img alt="image" width="111" src="https://user-images.githubusercontent.com/50552317/238328151-acb17f1b-533e-4d58-b678-146b0a6a0583.png"> Why the downvotes? It makes more sense than the darn "Forum" tag!
> 
> Because it's not necessary to tell the user that a reload is required until they actually toggle the addon or change a setting. And after they do that, some type of modal would be better than adding a tag.

OK that makes sense.

**Comment by iqnite** [comment]
Maybe we could do it like the "Reload the tabs" modal of TurboWarp?

**Comment by Samq64** [comment]
I agree something similar to TurboWarp is the best option. Are there any addons that need some settings to be marked as dynamic but not others?
